ARDemos/ARDemos/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  ARDemos
+//
+//  Created by Berta Devant on 11/01/2018.
+//  Copyright © 2018 Berta Devant. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
```

Comment:
this app delegate could be empty, we don't really need these methods, right?

Reply:
Done 7a5d800e410935e5ceca5c8022281e81651de2a5